src/main/java/com/epam/ta/reportportal/dao/ProjectRepositoryCustomImpl.java

```diff
@@ -94,4 +97,18 @@
 				() -> dsl.fetchCount(QueryBuilder.newBuilder(filter).build())
 		);
 	}
+
+	@Override
+	public int deleteByTypeAndLastLaunchRunBefore(ProjectType projectType, LocalDateTime bound, int limit) {
```

Comment:
the method name isn't compatible with logic in it?

Reply:
Why not? Propose better name.